airbyte-webapp/src/views/common/StoreProvider.tsx

```diff
@@ -14,7 +15,10 @@ const queryClient = new QueryClient({
 
 const StoreProvider: React.FC = ({ children }) => (
   <CacheProvider>
-    <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
+    <QueryClientProvider client={queryClient}>
+      <ReactQueryDevtools initialIsOpen={false} />
```

Comment:
Small nitpick. Could we make it `position="bottom-right"` so it's a bit more "out of the way" and not overlapping with the side navigation.